client/src/js/components/partials/messageItem.jsx

```diff
@@ -29,7 +25,10 @@ class MessageItem extends React.Component {
         <div className="right details">
             <p className="date right">
               {new Date(this.props.createdAt.substr(0, 10)).toUTCString().substr(0, 16)} &nbsp;
-              <i className="status icon ion-ios-checkmark-outline x15"></i>
+              { this.props.readBy.includes(this.props.username) ?
+                <i className="status icon ion-ios-checkmark x15"></i> :
+                <i onClick={this.props.updateReadBy(this.props.id)}
+                className="status icon ion-email-unread x15"></i> }
               <br/>
```

Comment:
@jattoabdul `Do not use <br> to increase the gap between lines of text; use the CSS margin property or the <p> element.` 
Source: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br